med_app_mobile/lib/pages/login_page.dart

```diff
@@ -0,0 +1,198 @@
+                            // Przycisk logujący do konta na podstawie hasła i e-maila
+                            margin: const EdgeInsets.fromLTRB(0, 20, 0, 0),
+                            decoration: ThemeHelper().buttonBoxDecoration(context),
+                            child: ElevatedButton(
+                              style: ThemeHelper().buttonStyle(),
+                              onPressed: () {
+                                Navigator.pushReplacement(
+                                  context,
+                                  MaterialPageRoute(
+                                    builder: (context) => const MainPage()
+                                  )
+                                );
+                              },
+                              child: const Padding(
+                                padding: EdgeInsets.fromLTRB(40, 10, 40, 10),
+                                child: Text(
+                                  'Login',
+                                  style: TextStyle(
+                                    fontSize: 20,
+                                    fontWeight: FontWeight.normal,
+                                    color: Colors.white,
+                                  ),
+                                ),
+                              ),
+                            ),
+                          ),
+                          Container(
+                            // Przycisk wysyłający do rejestracji, chyba nic nie trzeba dodawać
+                            margin: const EdgeInsets.fromLTRB(10, 15, 10, 20),
+                            alignment: Alignment.center,
+                            child: GestureDetector(
+                              onTap: () {
+                                Navigator.push(
+                                  context,
+                                  MaterialPageRoute(
+                                    builder: (context) => const RegistrationPage()
+                                  )
+                                );
+                              },
+                              child: Text(
+                                'Create account',
+                                style: TextStyle(
+                                  color: Theme.of(context).primaryColor,
+                                  fontSize: 18,
+                                  fontWeight: FontWeight.bold
+                                ),
+                              ),
+                            ),
+                          ),
+                          const Text(
+                            '--------------- Or sign in with ----------------',
+                            style: TextStyle(
+                              color: Colors.grey
+                            ),
+                          ),
+                          const SizedBox(width: 30),
+                          Container(
+                            // Przycisk do providera od Googla - oprogramować
```

Comment:
Tego typu rzeczy oznacza się słowem kluczowym TODO:
// TODO: add logic to Google login button

Łatwiej potem znaleźć i wygenerować zadania np. do Jiry